modules/nbde-compromise-of-key-material.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// security/nbde-implementation-guide.adoc
+
+[id="nbde-compromise-of-key-material_{context}"]
+= Rekeying compromised key material
+
+The compromise of individual key material on a Tang server, such as the physical theft of a Tang server or associated data, is cause to trigger the immediate rotation of keys. Specifically, perform the following actions:
+
+.Procedure
+
+. Rekey any Tang server holding the affected material.
+. Rekey all clients using the Tang server.
+. Destroy the original key material.
+
+Carefully assess any compromise of key material that might have led to the compromise of the master encryption key on any given node. Ideally, take the server offline and perform a full re-encryption of its disk. Reformatting and reinstalling on the same physical hardware, although taking longer in clock time, is easier and more rigorously automated and tested.
```

Comment:
```suggestion
. Scrutinize any incidents that result in unintended exposure of the master encryption key. If possible, take compromised nodes offline and re-encrypt their disks.  

[TIP]
Reformatting and reinstalling on the same physical hardware, although slow, is easy to automate and test.
```

This seemed like an embedded optional step and a TIP.

Reply:
Good rewrite! Changed.